diplomacy/web/package.json

```diff
@@ -16,12 +16,14 @@
     "bootstrap": "^4.3.1",
     "jquery": "^3.4.1",
     "lodash.clonedeep": "^4.5.0",
+    "lodash.get": "4.4.2",
```

Comment:
Allows to check for deep object paths and return a default value instead of thowing an error.

Example:
```
//defaults to 5:
var aNumber = get(myObj, 'deep.nested.objec.property.number', 5);
```

in contrast to

```
let aNumber;
try {
aNumber = myObject.deep.nested.objec.property.number;
} catch(e) {
aNumber = 5;
}
```

Or worse, retrieving deep object properties without error handling or other `get`-like utilities.

This is a must have for me in any modern js browser project (at least last time I checked in 2019 ha!). I would actually include the whole `lodash` package, but followed the pattern in this project instead (which is to include each lib function separately).

See https://lodash.com/docs/4.17.15